aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/curating_good_algorithm_listing.md

```diff
@@ -0,0 +1,46 @@
+
+### Highlights section
+| #  | Section    | Description                                                                                                      | Mandatory/Highly Recommended                                              | Sample Example                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|----|------------|------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| H1 | Highlights | Summarize algorithm performance metric                                                                           | Mandatory                                                                 | In benchmarks from the AutoGluon-Tabular paper, AutoGluon outperformed many popular open-source/commercial AutoML platforms on 50 classification/regression datasets from Kaggle/OpenML. AutoGluon is faster, more robust, and much more accurate than other tools, even outperforming the best-of five other AutoML platforms on most datasets. In two popular Kaggle competitions, AutoGluon beat 99% of the participating data scientists after just 4 hours of training. |
+| H2 | Highlights | Summarize Model performance metric+C:D                                                                           | Must-have for algo with pre-trained model. Highly recommended for others  | The model applying learning on top of an existing model trained on Pascal VOC 2007-2012 dataset  extended with 200 annotated images from the XXXX dataset in addition to 4000 augmented images to represent blur and foggy conditions. <br><br>The algorithm accepts customer data and fine tunes the base ml model further to achieve higher Mean average Precision (MaP) in short time.                                                                                    |
```

Comment:
`@ignorable`: do you want explicitly use the word "algorithm" here?

Reply:
Sure, we can use it